Resources/Prototypes/_RMC14/Entities/Structures/Windows/directional_windows.yml

```diff
@@ -11,12 +11,14 @@
       fix1:
         shape:
           !type:PhysShapeAabb
-          bounds: "-0.49,-0.49,0.49,-0.40"
+          bounds: "-0.49,-0.49,0.49,-0.38"
```

Comment:
Is this to make it draw properly when hugging against the windoor?

Reply:
Im not sure why i did this, I think it was to make the hitbox in line with the barricade